[2024-01-23T17:57:32.128+0200] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_to_redshift.load_to_redshift_table manual__2024-01-23T15:57:23.225495+00:00 [queued]>
[2024-01-23T17:57:32.206+0200] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_to_redshift.load_to_redshift_table manual__2024-01-23T15:57:23.225495+00:00 [queued]>
[2024-01-23T17:57:32.206+0200] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-23T17:57:32.224+0200] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): load_to_redshift_table> on 2024-01-23 15:57:23.225495+00:00
[2024-01-23T17:57:32.226+0200] {standard_task_runner.py:60} INFO - Started process 50252 to run task
[2024-01-23T17:57:32.230+0200] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_to_redshift', 'load_to_redshift_table', 'manual__2024-01-23T15:57:23.225495+00:00', '--job-id', '289', '--raw', '--subdir', 'DAGS_FOLDER/postgres_to_redshift.py', '--cfg-path', '/tmp/tmp70hky838']
[2024-01-23T17:57:32.231+0200] {standard_task_runner.py:88} INFO - Job 289: Subtask load_to_redshift_table
[2024-01-23T17:57:32.262+0200] {task_command.py:423} INFO - Running <TaskInstance: postgres_to_redshift.load_to_redshift_table manual__2024-01-23T15:57:23.225495+00:00 [running]> on host DESKTOP-9NHC01K.
[2024-01-23T17:57:32.321+0200] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='postgres_to_redshift' AIRFLOW_CTX_TASK_ID='load_to_redshift_table' AIRFLOW_CTX_EXECUTION_DATE='2024-01-23T15:57:23.225495+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-23T15:57:23.225495+00:00'
[2024-01-23T17:57:32.330+0200] {base.py:83} INFO - Using connection ID 'my_aws_cnn' for task execution.
[2024-01-23T17:57:32.330+0200] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='my_aws_cnn', conn_type='aws') credentials retrieved from login and password.
[2024-01-23T17:57:35.868+0200] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T17:57:35.873+0200] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=postgres_to_redshift, task_id=load_to_redshift_table, execution_date=20240123T155723, start_date=20240123T155732, end_date=20240123T155735
[2024-01-23T17:57:35.898+0200] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T17:57:35.908+0200] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
